redesign-project/src/styles.scss

```diff
@@ -0,0 +1,97 @@
+/* For desktop: */
```

Comment:
Note that the assignment is to start with a mobile-first stylesheet -- so you should be designing at phone sizes first, then increasing and adding media queries as you get larger.